controllers/burgers_controller.js

```diff
@@ -46,19 +43,17 @@ router.put("/api/burgers/:id", function(req, res) {
 
 router.delete("/api/burgers/:id", function(req, res) {
   var condition = "id = " + req.params.id;
-  console.log(req.params.id);
   
   if (req.params.id === 'all') {
     burger.truncate("burgers", function(result) {
-      console.log("result.affectedRows", result.affectedRows);
       if (result.affectedRows === 0) {
         // If no rows were changed, then the ID must not exist, so 404
         return res.status(404).end();
       } else {
         return res.status(200).end();
       }
     });
-  }
+  } else {
```

Comment:
Turns out, we needed this else statement.  The `return` statements in lines 51 & 53 just return you out of the callback on line 48.  They do return you out of the router.delete() method on 44.  The else prevents the burger.delete() function from also running :)